### PR TITLE
[feature] add help window feature when ? is pressed

### PR DESCRIPTION
**Pull request by theghostmac**

### CODE REVIEW

**Comment by radlinskii** [comment]
Hi, great job at adding the help window!

I have one suggestion, I think it should only be accessible Normal mode, so either during Pause, or before running the test.
Maybe also in the test results screen, but that can be a separate issue.

Guess the description wasn't clear enough, sorry about that.

**Comment by theghostmac** [comment]
Hey @radlinskii, I just addressed that in the last 2 commits.

Nice project btw.

**Comment by theghostmac** [comment]
Just did `cargo fmt` That should pass the `build` workflow.

**Comment by radlinskii** [comment]
the `build` job run is still failing,  let me know if you plan to fix it yourself or not

**Comment by radlinskii** [comment]
ref #70

**Comment by theghostmac** [comment]
I'll fix it. I've seen the issue.

**Comment by theghostmac** [comment]
Hi @radlinskii I've addressed the issue.

1. Changed the render method to take impl FrameWrapperInterface instead of Frame<B>
2. Removed the generic <B: Backend> since we're no longer using the raw Frame
3. Changed direct frame methods to use the FrameWrapperInterface methods
4. Simplified the imports since we don't need Backend anymore

**Comment by theghostmac** [comment]
Hey @radlinskii , 
I'm thinking of updating the `cli.rs` to not ignore the tests as the pass now.

**Comment by radlinskii** [comment]
did you even try to run the app with your changes?

File: src/help_window.rs [hunk 1]
```diff
@@ -48,16 +51,15 @@ impl HelpWindow {
...
         // Render block
         frame.render_widget(block, area);
 
         // Render text paragraphs
         for (i, &text) in help_text.iter().enumerate() {
             let paragraph = Paragraph::new(text).style(Style::default().fg(Color::White));
-            frame.render_widget(paragraph, chunks[i]);
+            if i < chunks.len() {
+                frame.render_widget(paragraph, chunks[i]);
+            }
```
Comment (radlinskii) [inline src/help_window.rs:62]: maybe consider simply creating the right amount of chunks, the same as the number of elements in help_text vector?

Comment (theghostmac) [reply src/help_window.rs]: done.

**Comment by radlinskii** [comment]
when help window is rendered then 's' keypress should not start test, or if it did, than the help window should hide.

**Comment by theghostmac** [comment]
Okay, the most user-friendly one is the econd option -- to automatically hide help when 's' is pressed.

The user can now:
- Toggle help with "?"
- Press 's' to dismiss help and start the test in one action
- Still use 'q' to quit whether help is shown or not

Any other changes required to improve the user experience?

**Review by radlinskii — APPROVED** [review]

**Pull request merged**